src/Consumption/QueueExtension.php

```diff
@@ -132,7 +133,7 @@ public function onPostMessageReceived(PostMessageReceived $context): void
      */
     public function onPreConsume(PreConsume $context): void
     {
-        $this->runtime = microtime(true) - $this->startedAt;
+        $this->runtime = $this->startedAt->diffInSeconds();
         $this->logger->debug(sprintf('Runtime: %s', $this->runtime));
```

Comment:
The unit for the time period should be shown.

Reply:
I added the time unit to the all log lines.